### ISSUE TITLE
normalize option nomenclature (hide/enable/set/etc.)

### ISSUE THREAD

**Issue by kylemcdonald** [openframeworks/openFrameworks#249]
ofHideCursor()/ofShowCursor()

ofEnableArbTex()
ofEnableSetupScreen()
ofEnableDataPath()
ofEnableAlphaBlending()
ofEnableSmoothing()

ofSetFullscreen() + ofToggleFullscreen()
ofSetVerticalSync()

etc.

**Comment by DomAmato** [comment]
Im scrubbing through the code base ill keep an eye out for naming consistencies

**Comment by bakercp** [comment]
@DomAmato let's get together soon!  All this fun stuff would be much better with friends and beer.

**Comment by DomAmato** [comment]
You have time this week? I will be on vacation the next two weeks but would certainly be game to knock some of these issue out as a tag team =)

**Comment by bakercp** [comment]
Yeah, later this week would be ideal let's rally Chicago crowd over on http://talk.olab.io.  You want to make a post?

**Comment by DomAmato** [comment]
sure ill make a post right now

**Comment by bilderbuchi** [comment]
@bakercp @DomAmato check out [this issue label](https://github.com/openframeworks/openFrameworks/labels/code-review). :-)

**Comment by bakercp** [comment]
@bilderbuchi nice tip -- as we don't have the authority to single (or dual) handedly make code-review issues, I think our focus is going to be on 0.9.0 bug fixes and documentation.

**Comment by bilderbuchi** [comment]
> as we don't have the authority to single (or dual) handedly make code-review issues, I think our focus is going to be on 0.9.0 bug fixes and documentation.

I'd think you _do_ have that (and it's just an issue label :-P) - it's just that that label has only been used to collect the output from that workshop, so I'm hesitant to tag more stuff with it (although it would make sense)

> I think our focus is going to be on 0.9.0 bug fixes and documentation.

this is a very sensible suggestion - let's get 0.9.0 out the door, and _then_ start nibbling away on other things!

**Comment by bakercp** [comment]
@bilderbuchi are you sure you can't forego child-rearing for a bit longer and keep herding the oF cats for a little bit longer post 0.9.0? :+1:

**Comment by bilderbuchi** [comment]
well, _that_ is not the reason I'm pushing for it. ;-) :P We shall see, I probably won't disappear completely...

**Comment by DomAmato** [comment]
[![Cat Herder](http://img.youtube.com/vi/m_MaJDK3VNE/0.jpg)](https://youtu.be/m_MaJDK3VNE "herding cats")

**Comment by oxillo** [comment]
Hi,

We are trying to clean out the many issues that we have.
Many versions of OF have been released since this was issued.

Can you help us by checking if this issue is still present in the latest version ?

In case of inactivity, we'll close this issue by end of january, 2020.

Thanks

**Comment by bakercp** [comment]
This is still an issue.  But could be combined into a larger issue about how OF will be refactored for 1.0.
